java/src/jmri/jmrit/logix/Engineer.java

```diff
@@ -64,7 +64,7 @@
 
     @Override
     public void run() {
-        log.debug("Engineer started warrant {}", _warrant.getDisplayName());
+        if (debug) log.debug("Engineer started warrant {}", _warrant.getDisplayName());
```

Comment:
These checks for ```debug``` need to be removed. The logging system can makes those checks in a very efficient manner (this used to be less true), can potentially be changed dynamically, so adding an additional set once test does nothing.

Where you feel that ```log.debug``` needs to be protected, protect it in a test for ```log.isDebugEnabled()```.

See http://www.slf4j.org/faq.html#logging_performance for more details.